charter.html

```diff
@@ -277,6 +277,11 @@ <h2>Coordination</h2>
           such that the group is encouraged to coordinate with the appropriate groups at the WHATWG, Ecma, and IETF as needed.
         </p>
 
+		<p>
+			The WG will coordinate with the Private Advertising Technology Community Group to solicit feedback from a wider community of interest. At any time, but always when a specification is undergoing a <a href="https://www.w3.org/2021/Process-20211102/#rec-track" title="The W3C Recommendation Track">state change</a>, the WG will ask the CG for feedback. The WG will allow the CG at least 4 weeks to gather that feedback.
+			<p>The goal of coordinating with the CG is to solicit feedback where meaningful change to the document from the last time feedback was gathered occurs.
+		</p>
```

Comment:
```suggestion
```

I realize that this is the goal, but I struggled with attempts at fixing the grammar here for a while before going with a "less is more" approach.